dev-docs/bidders/vibrantmedia.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: bidder
+title: Vibrant Media
+description: Vibrant Media Prebid Bidder Adapter
+biddercode: vibrantmedia
+gdpr_supported: true
+gvl_id: 7
+usp_supported: true
+media_types: banner, video, native
+safeframes_ok: false
+pbjs: true
+pbjs_version_notes: 6.4.0 and later
+---
+
+### Preliminaries
+
+The Vibrant Media Adapter requires set-up before being enabled. Please contact us at intellitxtsupport@vibrantmedia.com.
+
+Note: For video, we support outstream only.
+
+### Bid Params
+
+There are no Vibrant Media-specific bid params required, only Prebid.js-native ones (see examples below).
```

Comment:
Your first example shows a `placementId` parameter. The second example also shows you having a bidder-specific "video.skippable" and "video.playback_method". The third example shows you supporting a allowSmallerSizes parameter.

Please add the params table that every other adapter has listing out all the options the publisher needs to be aware of, including these.